test cases/common/149 special characters/meson.build

```diff
@@ -0,0 +1,21 @@
+project('ninja special characters' ,'c')
+
+python = import('python3').find_python()
+
+ret = run_command(python,
+  ['check_quoting.py',
+   'newline=
+',
+   'dollar=$',
+   'colon=:',
+   'space= ',
+   'multi= :$
+ :$
+'])
```

Comment:
This tests a string with two newlines separated by non-whitespace characters. My concern was that newlines and whitespaces might get 'squashed'. Could you check if two newlines in a row also works? Just adding another newline at the end of this should work, I think.